package.json

```diff
@@ -6,45 +6,47 @@
     "model-driven"
   ],
   "scripts": {
-    "build": "webpack",
+    "build": "rm -rf dist && webpack && cp package.json dist && cd dist && npm pack",
+    "build-windows": "(if exist dist rmdir dist /s /q) && webpack && copy package.json dist && cd dist && npm pack",
```

Comment:
Why do we need platform specific build?

Reply:
Because of the `rm` command, which is not working on Windows' `cmd`. The rest is platform-independent. There exist `rimraf` package, but for this, it can be overkill.